docs/installation/users.rst

```diff
@@ -90,34 +91,34 @@ Test your build
 If the installation is successful, you should be able to run the following test
 file in your terminal. Download the test file
 `here <https://raw.githubusercontent.com/deepanshs/mrsimulator-examples/master/test_file_v0.3.py?raw=true>`_
-or copy and paste the following code into a python file and run the code.
+or copy and paste the following code into a Python file and run the code.
 
 .. skip: next
 
 .. plot::
-    :caption: An example simulating solid-state NMR spectrums of static and MAS experiments
+    :caption: Simulation of static and MAS solid-state NMR spectra
 
     from mrsimulator import Simulator, SpinSystem, Site
     from mrsimulator.method.lib import BlochDecaySpectrum
     import matplotlib.pyplot as plt
 
     # Make Site and SpinSystem objects
-    H_site = Site(isotope="1H", shielding_symmetric={"zeta": 13.89, "eta": 0.25})
-    spin_system = SpinSystem(sites=[H_site])
+    H_site = Site(isotope = "1H", shielding_symmetric = {"zeta": 13.89, "eta": 0.25})
```

Comment:
@pjgrandinetti Just a quick note on standard Python formatting, `=` has a space on either side when assigning varibles like `foo = 3` but no spaces when used as a keword arguments for a function or a class like `bar = SomeClass(class_arg_1="baz")`

I've changed stuff on my end so it'll be fixed once I commit.

Reply:
i know this is a python convention, but I think it is a bad idea to do this in the documentation.   a space on both sides of the equal sign makes it easier for the reader
to see what's going on.  particularly for a non-Python coder.